### PR TITLE
Feat/user

### PR DESCRIPTION
**Pull request by josesrodriguez610**

### CODE REVIEW

**Review by scottanderson1130 — APPROVED** [review]

File: authentication/routes.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ authRouter.get(
   }
 );
 
+authRouter.get('/profile', (req, res) => {
+  res.send(req.user);
+  console.log(req.user);
```
Comment (jstrzesz) [inline authentication/routes.js]: remove console log

File: src/App.jsx [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
+    axios.get('/auth/profile')
+      .then((response) => {
+        if (response.data !== '') {
+          dispatch(setUser(response.data));
+          dispatch(setLogIn(true));
+        } else {
+          dispatch(setLogIn(false));
+        }
+      });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (jstrzesz) [inline src/App.jsx:30]: do not use es-lint disable

File: src/components/Nav.jsx [hunk 3]
```diff
@@ -12,9 +12,11 @@ import {
 } from '@material-ui/core';
 import { ChevronLeft, Menu } from '@material-ui/icons';
 import '../css/Nav.css';
+import { useSelector } from 'react-redux';
+import { selectLogIn } from '../features/userSlice.js';
 
 // eslint-disable-next-line react/prop-types
```
Comment (jstrzesz) [inline src/components/Nav.jsx:18]: do not use es-lint disable

Comment (josesrodriguez610) [reply src/components/Nav.jsx:18]: Changes have been made and will reflect in our next PR

**Review by ChristopherStumpe — APPROVED** [review]

**Pull request merged**